### ISSUE TITLE
0.2.0 release?

### ISSUE THREAD

**Issue by Be-ing** [udoprog/audio#10]
Rust 1.56 was released 3 November 2022 with GATs stabilized but this crate is still at 0.2.0-alpha.3. Is there anything left to do before releasing 0.2.0?

**Comment by udoprog** [comment]
Haven't found the time!

I would like to port a few project to make sure nothing obvious is missing. That is definitely something I could use some help with if you have any interesting candidates (like the latest version of rubato).

**Comment by Be-ing** [comment]
I got quite confused seeing big differences in the documentation between docs.rs and running `cargo doc` locally, then I realized that the last alpha tag was in April :sweat_smile: I think it'd help to tag another release at this point even if you call it another alpha.

**Comment by udoprog** [comment]
Will do!

When working on parallel changes, the recommended approach to try it out right now is by patching the project:
```toml
[patch.crates-io]
audio = { git = "https://github.com/udoprog/audio" } 
audio-core = { git = "https://github.com/udoprog/audio" } 
```

**Comment by udoprog** [comment]
I've published an `0.2.0-alpha.4` of `audio` and `audio-core` and yanked the previous alphas.
